### ISSUE TITLE
Internationalization

### ISSUE THREAD

**Issue by MorenoAlex** [omar/ByteSize#40]
Hi !
Amazing job on this library, it is so usefull.

Howerver I'm strugling will one thing : does the ToString function support units in other languages ?It does support english units (KB, MB, GB), but I'm having trouble with french.
In french a byte is called an "octet", so the units are writen like "Ko", "Mo", "Go"...
![image](https://user-images.githubusercontent.com/9295178/74230369-8a8cb680-4cc4-11ea-8e58-06779ef01a95.png)

I can do something like
`myByteSize.ToString("GB").Replace("GB", "Go")`
but it's not very clean.

Am I missing something ?
If not, any chance to see an internationalization feature in the future ?
Thanks :)

**Comment by elgonzo** [comment]
FYI: It's "ko" or "Kio", not "Ko" (https://fr.wikipedia.org/wiki/Octet) ;-)

**Comment by omar** [comment]
@NJullienSweet, yes I accept PRs and have accepted multiple ones already. The one in May has my feedback on it which needs to be addressed by the author before I merge it in.

If you have time to implement i18n, happy to review and merge it in.
